src/ReservoirDeployer.sol

```diff
@@ -69,8 +69,7 @@ contract ReservoirDeployer {
         // Set global parameters.
         factory.write("Shared::platformFee", DEFAULT_PLATFORM_FEE);
         factory.write("Shared::platformFeeTo", address(this));
-        // TODO: Is it okay to not set defaultRecoverer?
```

Comment:
let's just set it for now, cuz besides `recoverToken`, `skimExcessManaged` also relies on the recoverer. If we set it, it gives a more complete infra for testing

Reply:
There is an inconsistency in skimExcessManaged. It does not check the recoverer is set while recoverToken does check this...

However, we really want skimExcessManaged to never fail as its a safety hatch. This makes me think we should force recoverer to always be non-zero and then remove the recoverToken check?

Reply:
I realized that too, I'm okay with removing the check and we always make sure that there's a recoverer in the deploy scripts

Reply:
I guess this is the best we can do short of make `recoverer` a custom config type that cannot be set to zero. I'm okay with this

Reply:
okay check removed